recipe/meta.yaml

```diff
@@ -14,20 +14,20 @@ source:
         - skip_failing_test.patch  # [osx]
 
 build:
-    number: 3
+    number: 4
     skip: True  # [win]
 
 requirements:
     build:
-        - sqlite
+        - sqlite 3.9*
```

Comment:
To be honest I am not sure. All I can say is that `libspatialite` is not OK with the latest version. If another software is OK with both versions then this will will dictate the downgrade and things will be OK.